src/refda/exec.h

```diff
@@ -58,6 +60,14 @@ int refda_exec_run_seq(refda_exec_seq_t *seq);
  */
 void *refda_exec_worker(void *arg);
 
+/**
+ * Begin periodic execution of a time based rule
+ * @param[in] The agent context pointer
+ * @param[in] The rule to execute
+ * @return Non-zero if the rule could not be started
+ */
+int refda_exec_tbr_start(refda_agent_t *agent, refda_amm_tbr_desc_t *tbr);
```

Comment:
This could be renamed as `_enable` and used to control both the enabled state of the TBR and the bookkeeping for enabled rules.

Reply:
The original name was `_enable` :). But yeah, renamed and added start up book-keeping for TBR's. This is a good place to unify all of that.